### PR TITLE
Update SpringBoot@2.6.14

### PR DESCRIPTION
**Pull request by mcornaton**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+    id 'org.springframework.boot' version '2.6.14'
     id 'io.spring.dependency-management' version '1.1.0'
     id 'jacoco'
     id 'org.sonarqube' version '3.3'
     id 'maven-publish'
 }
 
 ext {
-    springCloudVersion = '2021.0.0'
+    springCloudVersion = '2021.0.5'
```
Comment (jbern0rd) [inline build.gradle:12]: Same question as for the SMS here

Comment (jbern0rd) [reply build.gradle:12]: Do we still need spring openfeign dependencies and spring-cloud-dependencies bom ?

Comment (mcornaton) [reply build.gradle:12]: This wouldn't work there. Removing Spring Cloud dependencies prevent the Result Proxy from starting:
```
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.aop.config.internalAutoProxyCreator': Initialization of bean failed; nested exception is java.lang.NoClassDefFoundError: org/aspectj/lang/annotation/Pointcut
```

**Review by jeremyjams — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (jbern0rd) [reply build.gradle:12]: Thanks, it is the spring retry mechanisms which is dependent on spring-cloud.

**Review by jbern0rd — APPROVED** [review]

**Pull request merged**